src/completion/completionProvider.ts

```diff
@@ -44,8 +45,9 @@ class CompletionProvider implements vscode.CompletionItemProvider {
                 const artifactIdHint: string = artifactIdNode && artifactIdNode.text ? artifactIdNode.text : "";
 
                 const centralItems: vscode.CompletionItem[] = await centralProvider.getGroupIdCandidates(groupIdHint, artifactIdHint);
+                const indexItems: vscode.CompletionItem[] = await indexProvider.getGroupIdCandidates(groupIdHint, artifactIdHint);

```

Comment:
Do you need a global indicator like `isJavaExtensionInstalled: boolean`, and calculate the items only when it's true?

Reply:
ok, I will add it.